README.md

```diff
@@ -43,6 +43,52 @@ MG serve to capture the **definitions** of terms introduced in the MIPs and MDs.
 
 
 ## Files and numbering
-Each MIP, MD or MG is stored in a separate subdirectory with the a name `mip-<number>`, `md-<number>` or `mg-<number>`. The subdirectory contains a README.md file that describes the MIP, MD, or MG. All assets related to the MIP, MD or MG are stored in the same subdirectory.
+Each MIP, MD or MG is stored in a separate subdirectory with the a name `mip-<number>`, `md-<number>` or `mg-<number>`. The subdirectory contains a `README.md` file that describes the MIP, MD, or MG. All assets related to the MIP, MD or MG are stored in the same subdirectory.
 
-MIPs, MDs and MGs are assigned their PR number as soon as they are drafted. MDs that do not introduce a new MIP are also accepted. Thus, there will be gaps in the MIP number sequence. These gaps will also emerge when MIPs are deprecated or rejected.
+MIPs start as Drafts. MIPs obtain a number during the review process described in [Status Terms](#status-terms). The number is the highest number in the MIP directory plus one.
+
+PRs that don't introduce a new MIP are also accepted. 
+- MIPs can be updated. PRs that update a MIP should state so in the PR title via `[Update] .... `. 
+
+There may be gaps in the numbering, as MIPs get rejected and removed.
+
+
+
+## Status Terms
+An MIP is proposed through a PR. Each MIP-introducing PR should have a status in the name in the form `[Status] ...`.
+
+An MIP should at all times have one of the following statuses:
+- **Draft** - (set by author) An MIP that is open for consideration. (It does not yet hold an MIP number)
+- **Review** - (set by author) The MIP is under peer review. The MIP should receive an **MIP number**, according to the rules described in the [Files and numbering](#files-and-numbering) section.
+
+>[!Note]
+> Currently the author has to understand from the PRs what the latest MIP number is. This is suboptimal and will be fixed by a later PR. 
+
+- **Accepted** - (set by editor) An MIP that has been accepted. All technical changes requested have been addressed by the author. There may be additional non-technical changes requested by the MIP editor.
+
+>[!Note]
+> In case the editors are not available for an unacceptable long period of time, a reviewer should assume the role of the editor interim. 
+
+After acceptance the MIP is merged into `main` and the branch should be deleted.
+
+Addtionally, the following statuses are used for MIPs that are not actively being worked on:
+- **Stagnant** - an MIP that has not been updated for 6 months.
+- **Withdrawn** - an MIP that has not been updated for 6 months.
+
+Finally, MIPs can also be updated
+- **Update** - (set by author) An MIP is being updated. The titel should list the MIP number, e.g. `[Update] MIP-0 ...`.
+
+
+## Editor
+
+Currently the editors are [@apenzk](https://github.com/apenzk). Volunteers are welcome.
```

Comment:
the intention was to invite company internal volunteers as opposed to external reviewers. i have removed the sentence as it was unclear.

Reply:
the editor can be bypassed as the rule remains that 3 approvals suffice. in which case the editor would have to apply [updates] if he is not satisfied. it is a non-blocking role. I have clarified this in the text.

Reply:
the point of [accepted] is that it gives the editor the chance to clean up the MIPs, ensure that MIP creators are added as code owners.

Reply:
I'm not convinced anymore that the editor role and list are needed. If we just consider merged MIP PRs to be **Accepted** then the list of merged PRs can be our Accepted "list". Draft PRs are drafts and open PRs are in review or stale. 

The weekly MIP ceremonies can help us develop a discipline as a team to focus on relevant MIPs. Or in general if an MIP is falling by the wayside, we can ping other team members to review it. I don't think the editor role or a separate list are needed for this functionality.

Reply:
i have reduced the role of editor as well as removed the accepted stage. still think that the drafts are important and should be easy to oversee and which they currently are not.. 

here is the example i draw from at Ethereum https://eips.ethereum.org/core

Reply:
It makes sense in that case, because they're using a separate website apart from GitHub. I think if this level of extra administration is not strictly necessary, then it should be avoided.

Reply:
thx for the feedback though.. it really helps to understand what is actually needed.

Reply:
@andygolay I would argue that because we will not be re-rerendering this content for a while, we should be taking extra care about its presentation natively.

Reply:
Agreed, with proper readability and quality standards in place that we're all aware of, it can be easy to implement.